test/functional/terminal/tui_spec.lua

```diff
@@ -379,7 +379,7 @@ describe('TUI', function()
   end)
 
   it('paste: normal-mode (+CRLF #10872)', function()
-    feed_data(':set ruler')
+    feed_data(':set ruler | echo')
```

Comment:
This change is needed because I removed `normal! a` which clears the command line.